build.gradle

```diff
@@ -27,6 +27,7 @@ dependencies {
     ext.guiceVersion = '5.0.1'
 
     implementation ("org.apache.jclouds:jclouds-core:${jcloudsVersion}")
+    implementation ("org.jfrog.buildinfo:build-info-extractor-gradle:4.21.0")
```

Comment:
@cristinabuenahora why this change?

Reply:
wouldn't compile otherwise - let me know if I was doing something wrong.

Reply:
@cristinabuenahora you should not need this line. The above change on line #4 should be enough to get things working.

Reply:
Ok, I'll remove and repush